### PR TITLE
Two MetaStation fixes

### LINKED ISSUE #18382: Metastation xenobiology kill chamber is broken.
Using the xenobiology kill chamber on metastation does literally nothing, as it isn't cold enough to kill any slimes.

### PR DESCRIPTION
**Pull request by ghost**
- Added bar stools to the MetaStation bar
- Fixed the MetaStation slime kill chamber not actually killing slimes and adds two vents to the chamber which are linked to a nearby freezer; also provided is a pair of winter boots and a winter coat (Fixes #18382)

Screenshots below (taken in Dream Maker).
_The revised slime euthanization chamber_
screenshot_1

_The modified barroom_
screenshot_2

### CODE REVIEW

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -93841,7 +93857,8 @@
 	start_active = 1
 	},
 /turf/open/floor/bluegrid{
```
Comment (xxalpha) [inline _maps/map_files/MetaStation/MetaStation.dmm]: These floors shouldn't be AI core floors, it makes no sense. Something like the kitchen's meat room floors would make more sense.

Comment (Xhuis) [inline _maps/map_files/MetaStation/MetaStation.dmm]: That wasn't up to me. Boxstation's kill room floors were bluegrid, and I wasn't the one who made the kill rooms originally, so I just used what Box used.

**Comment by Metacide** [comment]
Neat, thanks very much. :+1:

File: _maps/map_files/MetaStation/MetaStation.dmm (hunk 1 above)
Comment (xxalpha) [inline _maps/map_files/MetaStation/MetaStation.dmm]: I know.

**Pull request merged**